core/src/main/java/org/elasticsearch/index/translog/TranslogWriter.java

```diff
@@ -385,22 +385,6 @@ protected void readBytes(ByteBuffer targetBuffer, long position) throws IOExcept
         Channels.readFromFileChannelWithEofException(channel, position, targetBuffer);
     }
 
-    private static Checkpoint writeCheckpoint(
```

Comment:
why is this relevant? I rather not touch files that aren't relevant for the PR (feel free to push such a change as removal of dead code without a PR)